src/Components/NavBar/NavBarMobileMenu/NavBarMobileMenuAuthentication.tsx

```diff
@@ -55,6 +51,10 @@ export const NavBarMobileMenuLoggedIn: React.FC<NavBarMobileMenuLoggedInProps> =
         text: "Collector Profile",
         href: "/settings/edit-profile",
       },
+      isMyCollectionEnabled && {
+        text: "My Collection",
+        href: "/settings/my-collection",
+      },
```

Comment:
I believe that this might change the desktop view position of  "My Collection" tab